### PR TITLE
meraki_admin - Add documentation for tags and network access

### LINKED ISSUE #42686: meraki_admin - Add examples for networks and tags
##### SUMMARY
The `meraki_admin` module accepts `tags` and `networks` parameters but they aren't well documented. It should

* State they need to be passed as dicts
* Provide examples

##### ISSUE TYPE
 - Documentation Report

##### COMPONENT NAME
meraki_admin

##### ANSIBLE VERSION
```
2.7
```

### PR DESCRIPTION
**Pull request by kbreit**
##### SUMMARY
Documentation now lists examples for the tags and network dictionary structure.

Fixes #42686

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
meraki_admin

### CODE REVIEW

File: lib/ansible/modules/network/meraki/meraki_admin.py
Comment (samccann) [inline lib/ansible/modules/network/meraki/meraki_admin.py]: It wasn't clear to me what this new id and access referred to until I looked at the new examples. How about a line in the description that says something like "This list is a json dict with two parameters, C(id) and C(access)."  Also glancing at the code, it seems access can only be set to three choices. Would be worth adding that to this description as well.

File: lib/ansible/modules/network/meraki/meraki_admin.py
Comment (samccann) [inline lib/ansible/modules/network/meraki/meraki_admin.py]: This tag/access isn't defined in the description for tags. Can you add that information like you did for the json dict for networks?

File: lib/ansible/modules/network/meraki/meraki_admin.py
Comment (samccann) [inline lib/ansible/modules/network/meraki/meraki_admin.py]: Since both tags and networks can be a list, it would be helpful to make one of these examples show that (for example, showing two tags or two networks)

**Review by samccann — COMMENTED** [review]
Thanks for the PR!  It took me a little bit to understand the info you are adding here. I have a few comments that might help clarify things!

File: lib/ansible/modules/network/meraki/meraki_admin.py
Comment (dagwieers) [reply lib/ansible/modules/network/meraki/meraki_admin.py]: No, we have a construct for this already. Please use `suboptions`, check any of the modules using, like: *lib/ansible/modules/network/aci/mso_site.py*

**Review by dagwieers — CHANGES_REQUESTED** [review]
Using `suboptions` should render this properly in the docs.
https://docs.ansible.com/ansible/devel/modules/mso_site_module.html#parameters

File: lib/ansible/modules/network/meraki/meraki_admin.py
Comment (kbreit) [reply lib/ansible/modules/network/meraki/meraki_admin.py]: I've updated this so please take a look and let me know if what I have is more obvious.

File: lib/ansible/modules/network/meraki/meraki_admin.py
Comment (kbreit) [reply lib/ansible/modules/network/meraki/meraki_admin.py]: I moved the example from having one to two items.

File: lib/ansible/modules/network/meraki/meraki_admin.py
Comment (dagwieers) [inline lib/ansible/modules/network/meraki/meraki_admin.py]:
```suggestion
                - The privilege of the dashboard administrator on the network.
```

File: lib/ansible/modules/network/meraki/meraki_admin.py
Comment (dagwieers) [inline lib/ansible/modules/network/meraki/meraki_admin.py]:
```suggestion
                - Valid options are C(full), C(read-only), or C(none).
```

Comment (dagwieers) [reply lib/ansible/modules/network/meraki/meraki_admin.py]: I see some more without trailing dot.

**Review by dagwieers — DISMISSED** [review]

**Comment by kbreit** [comment]
@dagwieers Your suggestions were committed.

File: test/integration/targets/meraki_admin/tasks/main.yml
Comment (acozine) [inline test/integration/targets/meraki_admin/tasks/main.yml:59]: Change this one as well? 
```suggestion
        - tag: production
           access: read-only
```

Comment (dagwieers) [reply test/integration/targets/meraki_admin/tasks/main.yml:59]: For integration tests we do not care what formatting people use.
If this was the case, we would have a lot of work on our hands.

**Comment by kbreit** [comment]
I reverted back to before @acozine's suggestion.

**Comment by acozine** [comment]
@kbreit I didn't even realize I was requesting changes to a test file - thought I was still looking at docs.

**Comment by acozine** [comment]
rebuild_merge

File: test/integration/targets/meraki_admin/tasks/main.yml
Comment (acozine) [reply test/integration/targets/meraki_admin/tasks/main.yml:59]: You're right. I just saw the two similar blocks, one changed and one unchanged, and went into copy-editor mode.

**Comment by kbreit** [comment]
bot_status

**Comment by jctanner** [comment]
@kbreit @acozine @dagwieers the bot can not merge this because it has more than one committer. One of you will have to manually merge.

**Pull request merged**
